src/style.css

```diff
@@ -0,0 +1,126 @@
+.body {
+  margin: 0;
+  --color: #00acdc;
+  --star_size: 16px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card_text {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 16px 4px;
+}
+
+.card_product_code {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 16px 16px;
+}
+
+.stars_reviews {
+  display: flex;
+  margin: 0 16px 24px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: var(--star_size);
+  height: var(--star_size);
+}
+
+.stars__star:not(:last-child) {
```

Comment:
it is should be styles for any amount of active stars
![Screenshot from 2023-05-14 15-48-35](https://github.com/mate-academy/layout_product-cards/assets/33029864/3c96e94c-4065-48b2-9f06-313b746e9dc5)